header.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>Random text for header</p>
```

Comment:
shouldn't this be h1 tag and not p tag. please update.

Reply:
update